share/eds/blockstore.go

```diff
@@ -0,0 +1,195 @@
+package eds
+
+import (
+	"context"
+	"errors"
+	"sync"
+
+	"github.com/filecoin-project/dagstore"
+	"github.com/filecoin-project/dagstore/shard"
+	lru "github.com/hashicorp/golang-lru"
+	blocks "github.com/ipfs/go-block-format"
+	"github.com/ipfs/go-cid"
+	blockstore "github.com/ipfs/go-ipfs-blockstore"
+	ipld "github.com/ipfs/go-ipld-format"
+)
+
+var _ blockstore.Blockstore = (*Blockstore)(nil)
+
+var (
+	maxCacheSize            = 100
```

Comment:
is there a reason we are not being explicit about the type?

Reply:
Hey Matt, most of these comments apply to the PR that this is based on :)
Could you move them to #1232 and we can get them resolved there?

Reply:
The only new addition in this PR are files `ods.go`, and `ods_test.go`. Sorry, should have explicitly stated it in PR description